indigo_api/exporters.py

```diff
@@ -281,18 +281,28 @@ def get_notices(self, document, short=False):
         if short:
             # we only care if it's uncommenced and/or repealed for the running header
             if not work.commenced and work.repealed_date:
-                return _('Not commenced; Repealed')
+                return _('Not commenced') + '; ' + _(work.repealed_verb).capitalize()
```

Comment:
I hope that `work.repealed_verb` is always non-null if `repealed_date` is non-null. I know we can't enforce it in the db.